infrastructure/cli/auth/initializer.go

```diff
@@ -59,8 +59,10 @@ func (i *Initializer) Init() error {
 		return nil
 	}
 	if !currentConfig.AutomaticAuthentication() {
-		notification.Send(sglsp.ShowMessageParams{Message: "Please authenticate your Snyk user in order to start scanning"})
-		log.Info().Msg("Skipping authentication - automatic authentication is disabled")
+		if currentConfig.Token() != "" { // Only send notification when the token is invalid
+			notification.SendError(&AuthenticationFailedError{})
+		}
+		log.Info().Msg("Skipping scan - user is not authenticated and automatic authentication is disabled")
 		return errors.New(errorMessage)
```

Comment:
Why is it an error, if automatic authentication is disabled?

Reply:
Because the user doesn't have a valid token, so we cannot start a scan.
When the initializers return an error, the scan is skipped.

Reply:
Semantically it's not an error. It's just a normal case, when user is not authenticated. I would return `nil` here.